platform/app/public/config/default.js

```diff
@@ -293,305 +293,4 @@ window.config = {
   //       ))
   //   },
   // },
-  hotkeys: [
```

Comment:
Do you need an example somewhere of how to add a custom binding?  Probably something in e2e.js that demonstrates customizing the configuration bindings by adding a specific customization.